alertmanager_matrix.spec

```diff
@@ -0,0 +1,56 @@
+%define debug_package %{nil}
+
+%define mybuildnumber %{?build_number}%{?!build_number:1}
+
+Name:           alertmanager_matrix
+Version:        0.0.11
+Release:        %{mybuildnumber}%{?dist}
+Summary:        Service for sending alerts from the Alertmanager webhook to a Matrix room and managing Alertmanager.
+
+License:        EUPLv1.2
+URL:            https://github.com/Rudd-O/%{name}
+Source0:        https://github.com/Rudd-O/%{name}/archive/{%version}.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
These URLs should probably refer to this repo?

Reply:
Fixed.